web/cypress.config.ts

```diff
@@ -13,4 +13,6 @@ export default defineConfig({
       // implement node event listeners here
     },
   },
+  viewportWidth: 1280,
+  viewportHeight: 800,
```

Comment:
The size of the viewPort, so the portion of the page that actually appears on screen. I was thinking that the test is failing because the element was being clipped by the bottom of the page but it's still failing anyway, so i'll have to look into it further.